script/js_Ajax/index.js

```diff
@@ -0,0 +1,25 @@
+const API = 'https://jsonplaceholder.typicode.com';
+const btn = document.querySelector("button");
+const wraper = document.getElementsByClassName("wraper");
+
+btn.addEventListener("click", () => {
+    fetch(`${API}/users`)
+    .then(response => response.json())
+        .then(users => {
+            const divUSer = document.createElement('DIV');
+            const User_name = document.createElement('P');
+            const name = document.createElement('P');
+            const mail = document.createElement('P');
+
+            User_name.innerHTML = users.username;
```

Comment:
`users` is an array of objects. You need to access an element in the array first, and after that - an object property
f.e. `users[0].username`